docs/reference/cluster/reroute.asciidoc

```diff
@@ -103,3 +103,15 @@ are available:
     To ensure that these implications are well-understood,
     this command requires the special field `accept_data_loss` to be
     explicitly set to `true` for it to work.
+
+=== Retry failed shards
+
+In the case when the allocation of a shard failed multiple times without a
```

Comment:
The cluster will attempt to allocate a shard a maximum of
    `index.allocation.max_retries` times in a row (defaults to `5`), before giving
    up and leaving the shard unallocated. This scenario can be caused by
    structural problems such as having an analyzer which refers to a stopwords
    file which doesn't exist on all nodes.

    Once the problem has been corrected, allocation can be manually retried by
    calling the <<cluster-reroute,`_reroute`>> API with `?retry_failed`, which
    will attempt a single retry round for these shards.